srtcore/core.cpp

```diff
@@ -1367,7 +1367,7 @@ size_t srt::CUDT::fillHsExtKMRSP(uint32_t* pcmdspec, const uint32_t* kmdata, siz
         {
             m_RejectReason = SRT_REJ_IPE;
             LOGC(cnlog.Fatal, log << "createSrtHandshake: IPE: srtkm_cmd=SRT_CMD_KMRSP and no kmdata!");
-            return false;
+            return 0;
```

Comment:
In line about 1760 there's a call of this and the value 0 better be handled. It's an easy way out, just return false there.